skills-exercises/intro-video-skills/README.md

```diff
@@ -0,0 +1,28 @@
+# Instructions
+
+## **Submission**
+
+### Put the URL of the loom or youtube video here.
+
+### URL: https://www.youtube.com/watch?v=v-1G-DabuAs&feature=youtu.be
```

Comment:
Good experience in short span of time. One thing, please try to make eye contact in your intro.